File: src/main/java/nl/dannyjelsma/soundboardbot/commands/music/ShuffleCommand.java
package nl.dannyjelsma.soundboardbot.commands.music;

import com.jagrosh.jdautilities.command.Command;
import com.jagrosh.jdautilities.command.CommandEvent;
import nl.dannyjelsma.soundboardbot.audio.SoundManager;

public class ShuffleCommand extends Command {

    public ShuffleCommand() {
        name = "shuffle";
    }

    @Override
    protected void execute(CommandEvent event) {

        boolean currentShuffle = SoundManager.getInstance().shouldShuffle();

        SoundManager.getInstance().setShuffle(!currentShuffle);
        event.getChannel().sendMessage("Set shuffle to **" + !currentShuffle + "**.").queue();
    }
}

File: src/main/java/nl/dannyjelsma/soundboardbot/commands/music/ShuffleCommand.kt
package nl.dannyjelsma.soundboardbot.commands.music

import com.jagrosh.jdautilities.command.Command
import com.jagrosh.jdautilities.command.CommandEvent
import nl.dannyjelsma.soundboardbot.audio.SoundManager

class ShuffleCommand : Command() {
    init {
        name = "shuffle"
    }

    override fun execute(event: CommandEvent) {
        val currentShuffle: Boolean = SoundManager.shouldShuffle()

        SoundManager.setShuffle(!currentShuffle)
        event.channel.sendMessage("Set shuffle to **" + !currentShuffle + "**.").queue()
    }
}